packages/frontend/src/main/components/comments/CommentThreadViewer.vue

```diff
@@ -694,6 +694,13 @@ export default {
     window.addEventListener('beforeunload', async () => {
       await this.sendTypingUpdate(false)
     })
+
+    window.addEventListener('keydown', (e) => {
```

Comment:
Any global (e.g. window) event listeners we add should be explicitly removed, otherwise they remain even after this component is gone. And then on each creation of this component another event listener is added, eventually bogging the users device down with all of the duplicate event listeners.

Best practice is to remove all event listeners, when they're no longer necessary, even local ones on local DOM elements, but considering that the DOM is destroyed after the component is unmounted, it's possibly fine that we don't explicitly remove local event listeners.